image_classification/tensorflow/official/resnet/resnet_model.py

```diff
@@ -453,7 +453,8 @@ def __call__(self, inputs, training):
       inputs = tf.reshape(inputs, [-1, self.final_size])
       mlperf_log.resnet_print(key=mlperf_log.MODEL_HP_DENSE,
                               value=self.num_classes)
-      inputs = tf.layers.dense(inputs=inputs, units=self.num_classes)
+      inputs = tf.layers.dense(inputs=inputs, units=self.num_classes,
```

Comment:
For this line of change, do you know what would be the equivalent change for the resnet50-v1.5 in tf_cnn_benchmark? https://github.com/tensorflow/benchmarks/blob/master/scripts/tf_cnn_benchmarks/models/resnet_model.py Thx!